feature/home/src/main/java/com/yapp/feature/home/dialog/AttendanceDialog.kt

```
@@ -0,0 +1,174 @@
+                    modifier = Modifier.fillMaxWidth(),
+                    enable = true,
+                    text = stringResource(R.string.attendance_dialog_cancel_button),
+                    onClick = { onDismissRequest() }
+                )
+            }
+        }
+    )
+}
+
+@Composable
+private fun PinCodeInput(
```

detekt (1.23.7)

[warning] The function PinCodeInput appears to be too complex based on Cognitive Complexity (complexity: 30). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)